Userland/Utilities/aconv.cpp

```diff
@@ -63,13 +64,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     StringView input_format {};
     StringView output_format {};
     StringView output_sample_format;
+    u32 target_samplerate = 0;
 
     Core::ArgsParser args_parser;
     args_parser.set_general_help("Convert between audio formats");
     args_parser.add_option(input, "Audio file to convert (or '-' for standard input)", "input", 'i', "input");
     args_parser.add_option(input_format, "Force input codec and container (see manual for supported codecs and containers)", "input-audio-codec", 0, "input-codec");
     args_parser.add_option(output_format, "Set output codec", "audio-codec", 0, "output-codec");
     args_parser.add_option(output_sample_format, "Set output sample format (see manual for supported formats)", "audio-format", 0, "sample-format");
+    args_parser.add_option(target_samplerate, "Target samplerate", "target-samplerate", 's', "samplerate");
```

Comment:
Two things to note here in terms of CLI design:
- The target sample rate shouldn't have a short name, since it's a specialized feature that goes beyond just converting between two formats. As you can see, the same applies to the output sample format and the codec overrides.
- The name of this parameter should consistently be the "output" sample rate. "Target" is a specific term in the DSP context, but for the user, anything relating to the output has to be called "output" and anything relating to the input has to be called "input".